src/message_handler.cc

```diff
@@ -128,6 +128,22 @@ void EmitSemanticHighlighting(QueryDatabase* db,
           continue;  // applies to for loop
         kind = func->def->kind;
         detailed_name = short_name;
+
+        // Check whether the function name is acturally there.
+        // If not, do not publish the semantic highlight.
+        std::string concise_name(
+            detailed_name.substr(0, detailed_name.find('<')));
+        auto start_line = sym.loc.range.start.line;
```

Comment:
nit: no auto when type is not present in the same line

Reply:
If we specify the type here, it will not automatically change according to the original one?